app/models/user.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'users',
+    {
+      firstName: { type: DataTypes.STRING, allowNull: false },
+      lastName: { type: DataTypes.STRING, allowNull: false },
+      email: { type: DataTypes.STRING, allowNull: false, unique: true },
+      password: { type: DataTypes.STRING, allowNull: false },
+      createdAt: { type: DataTypes.DATE, allowNull: false },
+      updatedAt: { type: DataTypes.DATE, allowNull: false }
+    },
+    {
+      underscored: true
```

Comment:
Add `timestamps: true`, with this config you don't need to define createdAt and updatedAt in the model, `sequelize` set this values automatically for you. These fields should be in the migrations.

Put also ` tableName: 'users'` due to our models must be named in camelCase and the tables must be name in snake_case. With this configuration you say to the model what is the table associated.

Reply:
I found in the official sequalize docs that timestamps are set by default on "true", I removed the createdAt and updatedAt, nevertheless, I think adding timestamps true is unnecessary.
I also found that sequalize by default infers the table name if this is not given, and it expects it to be the plural of the model name (it even works with irregular plurals like person - people), in this case it can be inferred (model=user tableName=users)

Reply:
Ok with the timestamp. What happen if my table is named persons and my model is named users? It is an atypical case but it can occurs I prefer to say to sequelize the tables associated to the model to avoid future issues

Reply:
ok, done as requested